templates/admin/admin_list.html.twig

```diff
@@ -0,0 +1,40 @@
+{% extends 'admin/base_admin.html.twig' %}
+
+{%block heading %}  {% endblock %}
+
+{% block content %}
+    <div class="row " >
+        {% if entities %}
+            <div class="col-5" >
+                <div class="table-responsive pt-2">
```

Comment:
@daria-sieroshtan 
This class is often, ugh, troublesome.
I suggest you to remove it because your app anyway is designed for desktops only

Reply:
I prefer to keep it